charts/consul/values.yaml

```diff
@@ -1131,6 +1131,41 @@ server:
     # @type: string
     caCert: null
 
+  # Settings for potentially limiting timeouts, rate limiting on clients as well
+  # as servers, and other settings to limit exposure too many requests, requests
+  # waiting for too long, and other runtime considerations.
+  limits:
+    # Settings for enabling or disabling request limiting across all requests
+    # on each Consul server.
+    requestLimits:   
+      # Setting for disabling or enabling rate limiting.  If not disabled, it
+      # enforces the action that will occur when RequestLimitsReadRate
+      # or RequestLimitsWriteRate is exceeded.  The default value of "disabled" will
+      # prevent any rate limiting from occuring.  A value of "enforce" will block
+      # the request from processings by returning an error.  A value of
+      # "permissive" will not block the request and will allow the request to 
+      # continue processing.
+      # @type: string
+      mode: "disabled"
+      
+      # Setting that controls how frequently RPC, gRPC, and HTTP
+      # queries are allowed to happen. In any large enough time interval, rate
+      # limiter limits the rate to RequestLimitsReadRate tokens per second.
+      #
+      # See https://en.wikipedia.org/wiki/Token_bucket for more about token
+      # buckets.
+      # @type: integer
+      readRate: -1
```

Comment:
Why -1 instead of 100 which is the default?

Reply:
those are the defaults: https://github.com/hashicorp/consul/blob/main/agent/config/default.go#L106-L107

This documentation is incorrect.  I'll open a PR to correct it.

Reply:
Awesome! thanks1